winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
 ---
 layout: post 
 title: Content Types in WinUI AvatarView control | Syncfusion
-description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+The AvatarView control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing without the initials, custom image, or group view types.
```

Comment:
@MadhumithanJayapathi Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.

Reply:
@Manivannan-E Changed the content of the default as mentioned.